src/accounts/DeleteFreeAccountOverlay.tsx

```diff
@@ -77,15 +78,15 @@ const DeleteOrgOverlay: FC = () => {
       suggestions,
       reason: VariableItems[reason],
     }
-    event('DeleteOrgExecuted Event', payload)
+    event('DeleteAccountExecuted Event', payload)
 
```

Comment:
Same - I don't think we should change these event names unless analytics is okay with us effectively clearing the data.